### ISSUE TITLE
Multiaccount: Verified between

### ISSUE THREAD

**Issue by krille-chan** [krille-chan/fluffychat#189]
#### Description
Verified between two account on the same APP

#### To Reproduce
"Waiting on Verifed" on one account is over the "Emoji verified" from the other account - a click on "they match" would close the upper "Waiting on Verify" and cancle the verification

#### Additional information:
- Device: FP3
- OS and OS version: Android
- Installed version of FluffyChat: 0.42.2
- (Android only) Which store are you using: F-Droid
- (Android only) Are Google Services available:

**Comment by Mikaela** [comment]
Practically a duplicate of #145.
